master/buildbot/worker/base.py

```diff
@@ -296,7 +297,10 @@ def stopService(self):
 
     def startMissingTimer(self):
         if self.notify_on_missing and self.missing_timeout and self.parent:
-            self.stopMissingTimer()  # in case it's already running
+            try:
+                self.stopMissingTimer()  # in case it's already running
+            except error.AlreadyCalled:
```

Comment:
Okay, I'll try the suggestion with `is not None` or `missing_timer.active()` to avoid the `try..except` construct, whatever works.